src/content/docs/data-apis/custom-data/aws-privatelink.mdx

```diff
@@ -215,6 +215,19 @@ These are the New Relic endpoint services available via AWS PrivateLink:
             `com.amazonaws.vpce.us-east-1.vpce-svc-007e743510dc46024`
           </td>
         </tr>
+        <tr>
+          <td>
+
+          </td>
+          <td>
+            `identity-api.newrelic.com`
```

Comment:
@haus Is the first one supposed to be in the table cell above?

Reply:
i copied the blank cell from the us-east-2 version. i think it is there because these are part of the infrastructure product (the row above).

![image](https://github.com/user-attachments/assets/724a3873-40fa-4df3-b65b-4ae78e869cec)